src/main/kotlin/no/nb/bikube/newspaper/service/UniqueIdService.kt

```diff
@@ -0,0 +1,17 @@
+package no.nb.bikube.newspaper.service
+
+import jakarta.persistence.EntityManager
+import no.nb.bikube.core.util.logger
+import org.springframework.stereotype.Service
+
+@Service
+class UniqueIdService(
+    private val entityManager: EntityManager
+) {
+    fun getUniqueId(): String {
+        val query = entityManager.createNativeQuery("SELECT nextval('collections_id_seq')")
```

Comment:
```suggestion
        val query = entityManager.createNativeQuery("SELECT nextval('maxit.collections_id_seq')")
```
Vi hadde skjemaet som default, men tror ikke det er noe lurt for framtiden, så legg til schema

Reply:
Sikkert lurt, oppdatert